spec/lib/dispatch-rider/handlers/base_spec.rb

```diff
@@ -9,28 +9,32 @@ def process(options)
       "good job"
     end
   end
-  
+
   class ProcessWithImmediateRetry < DispatchRider::Handlers::Base
     def process(options)
       raise "I have failed!"
     end
-    
-    def retry_timeout; :immediate; end
+
+    def retry_timeout
+      :immediate
+    end
   end
-  
+
   class ProcessWithTenSecondRetry < DispatchRider::Handlers::Base
     def process(options)
       raise "I have failed!"
     end
-    
-    def retry_timeout; 10*60; end
+
+    def retry_timeout
+      10*60
+    end
   end
 
   describe "#do_process" do
-    let(:message){ double(:message, subject: 'some_message', body: { 'guid' => DispatchRider::Debug::PUBLISHER_MESSAGE_GUID }) }
+    let(:message) { double(:message, subject: 'some_message', body: {'guid' => DispatchRider::Debug::PUBLISHER_MESSAGE_GUID}) }
```

Comment:
Line is too long. [127/120]<br>Space inside { missing.<br>Space inside } missing.